motionplan/planManager.go

```diff
@@ -804,7 +850,7 @@ func (pm *planManager) generateWaypoints(request *PlanRequest, seedPlan Plan, wp
 		if err != nil {
 			return nil, err
 		}
-		waypoints = append(waypoints, atomicWaypoint{mp: pathPlanner, startState: from, goalState: to})
+		waypoints = append(waypoints, atomicWaypoint{mp: pathPlanner, startState: from, goalState: to, origGoal: true})
```

Comment:
also weird to me that we would always be setting this to true

Reply:
Ack! Good catch! This should not always be true.